annotated_types/test_cases.py

```diff
@@ -128,7 +128,7 @@ def cases() -> Iterable[Case]:
     yield Case(at.IsDigit[str], ['123'], ['', 'ab', 'a1b2'])
     yield Case(at.IsAscii[str], ['123', 'foo bar'], ['£100', '😊', 'whatever 👀'])
 
-    yield Case(Annotated[int, at.Predicate(lambda x: x % 2 == 0)], [0, 2, 4], [1, 3, 5])
+    yield Case(Annotated[int, at.Predicate(lambda x: x % 2 == 0)], [0, 2, 4], [1, 3, 5])  # type: ignore
```

Comment:
These now give a type error because the type of `x` is unknown so you get `Predicate[Unknown]`